src/textures/TextTexture.mjs

```diff
@@ -428,7 +429,9 @@ export default class TextTexture extends Texture {
 
         return function(cb) {
             const canvas = this.stage.platform.getDrawingCanvas();
-            const renderer = new TextTextureRenderer(this.stage, canvas, args);
+            const renderer = Utils.isWeb ?
```

Comment:
Is there a way to detect spark platform specifically? E.g. by checking sparkscene object in global namespace, but maybe there is a better way?

Reply:
Sure, i would be happy to add a way to detect the Spark platform specifically.  Does something like the following look okay to you?

`
(this.stage.platform.platformName && typeof this.stage.platform.platformName === "function" && this.stage.platform.platformName() === "spark")
`

If not, can you please suggest a check that would be best?  Also, do you have a preference if I should encapsulate this check somewhere?

Reply:
More like, how can application detect if it runs on spark platform? Is there e.g. global object avaliable only on spark, or perhaps there is spark specific userAgent?
Current implementation (although very crude) gives something like this:

```javascript
Utils.isNode = (typeof window === "undefined");
Utils.isWeb = (typeof window !== "undefined");
Utils.isWPE = Utils.isWeb && (navigator.userAgent.indexOf("WPE") !== -1)
```

Some of those probably need to be altered to account for spark aswell.

Reply:
Currently, Spark doesn't have a navigator object (similar to node).  We can use the same check as we have for node `(typeof window === "undefined")`, but do you have have a suggestion how to best differentiate between node and Spark?

Reply:
Or (if you don't have a preference), I can add something to the platform area to differentiate between node and Spark.  Let me know and I will update.

Reply:
Can sparkscene object be used to determine platform? I didn't see it imported anywhere so I assume it's global object specific to spark environment?

Reply:
Yeah, great idea!  I'll add that as part of the check.

Reply:
Added a Utils.isSpark with the check suggested above.